### PR TITLE
Added .kill() method to remove hero from game upon contact with enemy.

### PR DESCRIPTION
**Pull request by hotblack86**

### CODE REVIEW

**Review by CKKH — APPROVED** [review]
Nice find!

**Review by Sindex42 — APPROVED** [review]

File: src/hero.py [hunk 1]
```diff
@@ -52,7 +52,9 @@ def collide_with_enemy(self, d_x=0, d_y=0):
         ''' Check for enemy collision '''
         for enemy in self.game.all_sprites:
             if enemy.x_pos == self.x_pos + d_x and enemy.y_pos == self.y_pos + d_y:
-                print("Enemy collision")
+                print("Game Over!")
```
Comment (michaelnguyen974) [inline src/hero.py:55]: Simple and clean

**Review by michaelnguyen974 — APPROVED** [review]

**Review by matharotheelf — COMMENTED** [review]
Good work

**Review by matharotheelf — APPROVED** [review]
Good work

**Pull request merged**